GUI/app/tabs/thermal_view_control.py

```diff
@@ -109,8 +110,13 @@ def update_canvas(self, pixels: List[int]) -> None:
             for p in pixels
         ]
 
+        # Rotate 90° to orient for mounting correctly
+        float_pixels_matrix = np.reshape(float_pixels, (8,8))
```

Comment:
use `self.camera_x` and `self.camera_y` instead, so just there are less magic numbers

Reply:
Otherwise, lgtm

Reply:
noted ill change that now.